lighthouse-core/gather/gatherers/dobetterweb/document-event-listeners.js

```diff
@@ -115,17 +127,24 @@ class PageLevelEventListeners extends Gatherer {
   }
 
   afterPass(options) {
+    const driver = options.driver;
+
     return this.unlistenForScriptParsedEvents(options.driver)
-      .then(_ => this.collectListeners(LISTENER_LOCATIONS))
-      .then(listeners => {
-        this.artifact = listeners;
-      }).catch(_ => {
-        this.artifact = {
-          usage: -1,
-          debugString: 'Unable to gather passive events listeners usage.'
-        };
-      });
+        .then(_ => driver.sendCommand('DOM.enable'))
```

Comment:
the general rule is that you need to enable() a domain if you want to receive events from it.
if you're just calling methods on it, then no need to enable.
(and the catch is that sometimes calling disable() will undo a setting that you did with a method (like network throttling))

so given that, i'd recommend you nuke the enable/disable